internal/app/service.go

```diff
@@ -733,3 +736,47 @@ func (svc *Service) setServiceKey(profile string) {
 func (svc *Service) BuildContext(correlationId string, contentType string) interfaces.AppFunctionContext {
 	return appfunction.NewContext(correlationId, svc.dic, contentType)
 }
+
+// Publish pushes data to the messagebus
+func (svc *Service) Publish(v any) error {
+	messageClient := bootstrapContainer.MessagingClientFrom(svc.dic.Get)
+	if messageClient == nil {
+		return fmt.Errorf("messagebus is disabled via configuration")
```

Comment:
Create a constant for this message since it is used in 4 places. 

```suggestion
		return fmt.Errorf("publish failed due to MessageBus disabled via configuration")
```